site/docs/v1/tech/migration-guide/general.adoc

```diff
@@ -581,6 +581,58 @@ You considered this situation while planning your migration, but now, implement
 
 You can mix and match these approaches. For example, you could migrate all paying customers, even those who haven't signed in during the migration period. At the same time you could archive the data of free accounts; those potential customers may have been trialing your application and may even have forgotten they have an account.
 
+==== The Forgot Password Use Case
+
+What happens when a user forgets their password and attempts to reset it, but you are in the middle of a slow migration? There are two scenarios:
+
+* The user has been migrated and has an account in FusionAuth
+* The user has not been migrated and still only has an account in the legacy system
+
+In either case, when a user submits a forgot password request, FusionAuth will display a message similar to `We've sent you an email containing a link that will allow you to reset your password. Once you receive the email, follow the instructions to change your password.` 
+
+An email will only be sent in the first case, where the user exists in FusionAuth. In the second case, the user will never receive an email.
+
+Why, then, is this message displayed for both sets of users? 
+
+The answer is that FusionAuth *can't distinguish* between users who have not been migrated and users that don't exist. In order to secure your user data and prevent enumeration attacks, FusionAuth doesn't reveal whether there is a valid account for a given username or email.
+
+If FusionAuth were to display a message such as `There is no account in the system` for a user who had not been migrated, attackers could determine whether an account existed, by trying different values and noting the different error messages.
+
+This unfortunately means that the "Forgot Password" experience isn't as smooth during a migration as it is afterwards.
+
+The current recommendation is to update the error message to something like:
+
+`We've sent you an email containing a link that will allow you to reset your password. Once you receive the email follow the instructions to change your password. If you don't receive an email, please check your spam folder or contact customer service.`
+
+Make sure you provide a link to a form, phone number or other means of contacting customer service. Customer service representatives can then check to see whether the user is migrated or not. 
+
+They may be able to trigger a password reset email from the appropriate user data store. You can do that from within the FusionAuth administrative user interface.
```

Comment:
Is this referring to linking back to the legacy system, and triggering a Forgot Password workflow?

It reads as though the User is migrated to FusionAuth, however, I thought the use-case was when a user is pre-migration, and may have forgotten their password (unable to Slow Migrate via Connector to FA).

Reply:
Good point, I reworked to be clearer. The point is that if the user calls in the customer service rep can check to see which system they are in and reset the password in the appropriate one.